ocaml/xapi/storage_smapiv1_wrapper.ml

```diff
@@ -453,6 +453,9 @@ functor
         List.fold_left perform_one vdi_t ops
 
       let perform_nolock context ~dbg ~dp ~sr ~vdi ~vm this_op =
+        debug "perform_nolock dp=%s, sr=%s, vdi=%s, vm=%s, op=%s" dp
```

Comment:
I would always include `__FUNCTION__` in logs to understand where a log line is coming from.

Reply:
accept

Reply:
done